pyttb/tensor.py

```diff
@@ -36,19 +36,12 @@ class tensor:
     data: np.ndarray
```

Comment:
I do not think we want these class variables. now that we have move our constructor into `__init__`. I think that `self.data` and `self.shape` defined in `__init__` makes them instance variables, which is what we want. I don't think I fully understand this, but it seems that if we define instance variables in `__init__` that have the same name as the class variables (as defined here), we never access the class variable -- thus it is not needed.

Reply:
Ya that's a good catch. I don't think these should have ever been class variables and I was just using them when I was fighting adding typing.